cscs-checks/tools/profiling_and_debugging/intel_advisor_roofline.py

```diff
@@ -0,0 +1,161 @@
+import os
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.required_version('>=2.14')
+@rfm.parameterized_test(*[[repeat, toolsversion, datalayout]
```

Comment:
Do  you need to add further values to repeat and toolsversion? If not, I suggest putting the values directly instead of using a `for` loop.

Reply:
see https://github.com/eth-cscs/reframe/pull/721#discussion_r270182089 
and also in the future it will be needed.